app/views/session/delete_answers.scala.html

```diff
@@ -0,0 +1,40 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.ViewConfig
+
+@this(
+    mainTemplate: views.html.main_template,
+    viewsHelpers: views.ViewsHelpers,
+    viewConfig: ViewConfig
+)
+
+@()(implicit request: Request[_])
+
+@import viewsHelpers.requestSupport._
+@mainTemplate(
+    title =s"${LangMessages.`For your security, we signed you out`.show} - GOV.UK",
+//    navTitle = Messages("delete-answers.heading"),
+    headerTitle = "header title",
```

Comment:
This header title looks unfinished - should it take a pre-defined message from `LangMessages`?